src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -355,7 +367,7 @@ def init_sync_d_lag_tables(db_conn):
         return lag_name_if_name_map, if_name_lag_name_map, oid_lag_name_map, lag_sai_map
 
     lag_sai_map = db_conn.get_all(COUNTERS_DB, b"COUNTERS_LAG_NAME_MAP")
-    lag_sai_map = {name: sai_id.lstrip(b"oid:0x") for name, sai_id in lag_sai_map.items()}
+    lag_sai_map = {name: get_sai_id_key(db_conn.namespace, sai_id.lstrip(b"oid:0x")) for name, sai_id in lag_sai_map.items()}
```

Comment:
>lag_sai_map [](start = 4, length = 11)

This line is too long to read. Could you rewrite by a loop?

Reply:
updated as per comment.